modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7_{context}"]
+= {mtc-full} 1.7 release notes
+
+[id="new-features-and-enhancements-1-7_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* The {mtc-full} ({mtc-short}) operator now depends upon the OpenShift API for Data Protection (OADP) operator. When you install the {mtc-short} operator, the Operator Lifecycle Manager (OLM) automatically installs the OADP operator in the same namespace.
+
+* You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters using the `crane tunnel-api` command.
```

Comment:
This sounds like it should be "by using." Per the ISG:
> ! using noun
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

Reply:
Fixed.